buildah_intro/buildah_intro.sh

```diff
@@ -32,18 +38,18 @@ intro() {
 
 version() {
     # Buildah version inside a container
-    echo "Check the Buildah version"
+    echo "${blue}Check the Buildah version${reset}"
```

Comment:
Could you generate a command 
colorecho
and colorread
Do to this rather then having to do the commands all over the place.